### ISSUE TITLE
Spearhead Navigation Block: Text/link color does not change

### ISSUE THREAD

**Issue by eduardozulian** [Automattic/themes#3686]
## Steps to replicate

1. Activate the theme
2. Add the Navigation Block
3. Change the text color in the block settings

## Result

Text Color option does not work on both editor and front-end. It only changes non-link elements like the sub-menu icon:

<img width="865" alt="Screen Shot 2021-04-20 at 13 47 23" src="https://user-images.githubusercontent.com/1136160/115623763-02467000-a2d0-11eb-9711-d4f6a024f071.png">

## Expected
To see colors changing.

<sub>Part of the Empathy Flow Testing p9b1hy-1Fq-p2</sub>

**Comment by formosattic** [comment]
Ran into a similar case (4859659-zd-woothemes) with the Heading, Paragraph block.

Colors & Backgrounds are set and might be overwriting the custom settings in the editor, similar to https://github.com/Automattic/themes/issues/5567

Workarounds:

* Text color: use Highlight
* Text background: use Group block with custom background color

**Comment by jffng** [comment]
Hi there, thanks for the report and follow up. 

I tested this locally and on dotcom with Spearhead and am able to change the navigation block's text color, so it looks like the original issue has been resolved.

I looked at the happychat report which is quite the same issue, and verified that I can set custom colors via the customizer with Spearhead. While the palette shown in the editor does not reflect those custom colors, the blocks appear as the correct customized colors in the editor and front.

Going to close this one.
